src/main/java/core/basesyntax/strategy/OperationService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationService {
+    void processOperation(FruitTransaction fruitTransaction);
+}
```

Comment:
it's better to rename it to OperationHandler and do the same for its implementations, this way will keep our naming consistent and stick to the strategy pattern